plenum/common/monitor_strategies.py

```diff
@@ -57,16 +59,24 @@ def remove_instance(self, inst_id):
         self._instances.remove(inst_id)
 
     def reset(self):
+        logger.info("Resetting accumulating monitor")
+        self._input_txn_rate.reset(self._timestamp)
         self._alert_timestamp = {inst: None for inst in self._instances}
         self._ordered.clear()
 
     def update_time(self, timestamp: float):
         self._timestamp = timestamp
         self._input_txn_rate.update_time(timestamp)
         for inst_id in self._instances:
-            if not self._is_degraded(inst_id):
+            is_alerted = self._alert_timestamp[inst_id] is not None
```

Comment:
Can it throw `KeyError` in case an instance was added? `add_instance` doesn't add a value for new instance to `_alert_timestamp`.

Reply:
Now it won't throw, also covered by test